.github/workflows/maven.yml

```diff
@@ -43,17 +43,21 @@ jobs:
       with:
         java-version: 21
         distribution: 'temurin'
-            
+
+    - uses: actions/setup-node@v4
+      with:
+        node-version: "18"
```

Comment:
Should we move to the current LTS (v20) here?

Reply:
We decided to go with the older version, to support devs that still use this version